specification/sql/resource-manager/Microsoft.Sql/preview/2017-03-01-preview/restorePoints.json

```diff
@@ -173,6 +173,50 @@
             "$ref": "./examples/DatabaseRestorePointsGet.json"
           }
         }
+      },
+      "delete": {
+        "tags": [
+          "RestorePoints"
+        ],
+        "description": "Deletes a restore point.",
+        "operationId": "RestorePoints_Delete",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ResourceGroupParameter"
+          },
+          {
+            "$ref": "#/parameters/ServerNameParameter"
+          },
+          {
+            "$ref": "#/parameters/DatabaseNameParameter"
+          },
+          {
+            "name": "restorePointName",
+            "in": "path",
+            "description": "The name of the restore point.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
```

Comment:
is delete long-running, i.e. returns 202 + operationresults? Or does it complete within <5 seconds?

Reply:
Cool, just checking we do not need to worry about more timeouts :)

Reply:
so yes, <5 seconds